package.json

```diff
@@ -33,7 +33,7 @@
   "license": "MIT",
   "dependencies": {
     "@truffle/config": "1.2.33",
-    "@truffle/hdwallet-provider": "1.1.1",
+    "@truffle/hdwallet-provider": "^1.1.1",
```

Comment:
We need to be so carefull with the ^ versions, we need to bring support to all the future changes on it.